modules/virt-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -6,16 +6,19 @@
 [id="virt-troubleshooting-incorrect-policy-config_{context}"]
 = Troubleshooting an incorrect node network configuration policy configuration
 
-You can apply changes to the node network configuration across your entire cluster by applying a node network configuration policy.
-If you apply an incorrect configuration, you can use the following example to troubleshoot and correct the failed node network policy.
+You can apply changes to the node network configuration across your entire cluster by applying a node network configuration policy. 
 
-In this example, a Linux bridge policy is applied to an example cluster that has three control plane nodes and three compute nodes.
-The policy fails to be applied because it references an incorrect interface.
-To find the error, investigate the available NMState resources. You can then update the policy with the correct configuration.
+If you applied a wrong configuration, you can use the following example to troubleshoot and correct the failed node network policy. The example attempts to apply a Linux bridge policy to a cluster that has three control plane nodes and three compute nodes. The policy is not applied because the policy references the wrong interface. 
```

Comment:
```suggestion
If you applied an incorrect configuration, you can use the following example to troubleshoot and correct the failed node network policy. The example attempts to apply a Linux bridge policy to a cluster that has three control plane nodes and three compute nodes. The policy is not applied because the policy references the wrong interface. 
```